server/channels/app/emoji.go

```diff
@@ -366,3 +366,17 @@ func (a *App) deleteReactionsForEmoji(emojiName string) {
 		mlog.Warn("Unable to delete reactions when deleting emoji", mlog.String("emoji_name", emojiName), mlog.Err(err))
 	}
 }
+
+func (a *App) confirmEmojiExists(c request.CTX, emojiName string) *model.AppError {
```

Comment:
Was the plan to re-use this function in the other places where we set emojis?
If so does it make sense to change that here, or in a follow-up PR?

Reply:
It's only useful where we want to verify that an emoji exists with a given name, so we could potentially use it in the PR you did to confirm an emoji exists when making a reaction, but I wasn't sure if that was in yet. I can't think of anywhere else I'd want to use it right now

Reply:
I'm good to use it in my PR, it does seem useful.